### ISSUE TITLE
[TablePagination] - Customizing text "XXX of YYY" after rowsPerPageLabel

### ISSUE THREAD

**Issue by MichalWadowski** [mlaursen/react-md#742]
In TablePagination we can customize text of 'rowsPerPageLabel'. But right to SelectField is placed label e.g. "1-10 of 759" and we can't customize it. Most important is the word "of", which cant be customized for other languages.

**Comment by mlaursen** [comment]
Oof, that's right. I think I'll add a `paginationFormat` prop (still thinking of a name) that will be a function that'll be:
```ts
declare function paginationFormat({ start: number, pageSize: number, total: number }): string
```

so that you can manually create that string on your own. It'll have a default implementation of what it is now.

**Comment by mlaursen** [comment]
This was actually fixed in the 1.10.0 release with the introduction of the `paginationLabel` prop.
